core/payment-driver/zksync/src/zksync/utils.rs

```diff
@@ -45,9 +44,9 @@ pub fn pack_up(amount: &BigUint) -> BigUint {
 
 fn increase_least_significant_digit(amount: &BigUint) -> BigUint {
     let digits = amount.to_radix_le(10);
-    for i in 0..(digits.len()) {
+    for i in 0..digits.len() {
         if digits[i] != 0 {
-            return amount + pow(BigUint::from(10u32), i);
+            return amount + BigUint::from(10u32).pow(i.try_into().unwrap());
```

Comment:
`pow` was the only reason why `num` was imported over `num-bigint`

any idea's how to remove this try_into? usize -> u32 should always be safe AFAIK

Reply:
Just use `i as u32`

Reply:
that works, thanks :D